src/mudlet.cpp

```diff
@@ -61,17 +61,23 @@
 #include <QJsonObject>
 #include <QJsonValue>
 #include <QNetworkDiskCache>
+#include <QMediaPlayer>
 #include <QMessageBox>
 #include <QNetworkDiskCache>
 #include <QScrollBar>
+#include <QShortcut>
 #include <QTableWidget>
 #include <QTextStream>
+#include <QTimer>
 #include <QToolBar>
+#include <QToolButton>
 #include <QVariantHash>
 #include <QRandomGenerator>
 #include <zip.h>
 #include "post_guard.h"
 
+using namespace Mudlet;
```

Comment:
Nothing much, as it turns out. I thought there'd be more instances of global code in `mudlet` that doesn't need to be in the class itself because it doesn't access any of its variables, and I didn't clean that up..

Another case for some `util.h`/`util.cpp` class to collect random stuff that doesn't belong in any classes?

Reply:
There are some `static` functions in this class that are there partly because some many classes include this one's header I guess, not an ideal arrangement I suppose but it was a quick (though *not entirely clean*) way to put some utility functions in...

Reply:
I removed that namespace for now, can be re-visited later.